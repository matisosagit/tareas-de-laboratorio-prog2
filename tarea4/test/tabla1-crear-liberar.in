# creo una tabla de tamanio 4
crearTabla 4

# libero la tabla
liberarTabla

# creo una tabla de tamanio 15
crearTabla 15

# libero la tabla
liberarTabla

Fin