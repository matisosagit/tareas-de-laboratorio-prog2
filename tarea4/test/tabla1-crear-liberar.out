1> #  creo una tabla de tamanio 4.
2> Tabla de fichas de vacunación con cantidad estimada 4 creada.
3> #  libero la tabla.
4> Se liberó la tabla de fichas de vacunación.
5> #  creo una tabla de tamanio 15.
6> Tabla de fichas de vacunación con cantidad estimada 15 creada.
7> #  libero la tabla.
8> Se liberó la tabla de fichas de vacunación.
9> Fin.
